models/preles/R/runPRELES.jobsh.R

```diff
@@ -18,6 +18,10 @@
 ##' @author Tony Gardella, Michael Dietze
 runPRELES.jobsh <- function(met.file, outdir, parameters, sitelat, sitelon, start.date, end.date) {
   
+  if(!require("Rpreles")){
+    logger.severe("The Rpreles package is not installed. 
+                  Please contact the PEcAn team to obtain source code")
```

Comment:
likewise, provide link to documentation and/or instructions on HOW to contact the PEcAn team. That said, if we have permissions to redistribute we should just provide a weblink. If we don't have permissions, they should be contacting the model team not us.